src/script/ui/app/toolbar.jsx

```diff
@@ -169,7 +169,7 @@ const template = [
 		component: TemplatesList
 	},
 	'template-lib',
-	'chiral-flag'
+	'enhanced-stereo'
```

Comment:
Please read in details #56 , may be we should move button to top toolbar or somewhere else? Not sure that enhanced stereo tool should be on the same place where was chiral-flag tool

Reply:
We agreed to do it later